### PR TITLE
[SW-865] Fix name claim

### PR DESCRIPTION
**Pull request by CedrikNikita**
missed in #1976

### CODE REVIEW

**Review by pstachel — APPROVED** [review]

File: src/popup/pages/Names/Claim.vue [hunk 1]
```diff
@@ -72,94 +73,99 @@ import CheckBox from '../../components/CheckBox.vue';
...
-            this.$store.commit('names/setPendingAutoExtendName', name);
+          const { salt } = await sdk.aensPreclaim(fullName);
+          claimTxHash = (await sdk.aensClaim(fullName, salt, { waitMined: false })).hash;
+          if (autoExtend.value) {
+            root.$store.commit('names/setPendingAutoExtendName', fullName);
           }
-          this.$router.push({ name: ROUTE_ACCOUNT_DETAILS_NAMES });
-        } catch (e) {
+          root.$router.push({ name: ROUTE_ACCOUNT_DETAILS_NAMES });
+        } catch (e: any) {
```
Comment (subhod-i) [inline src/popup/pages/Names/Claim.vue:125]:
```suggestion
        } catch (e: Exception) {
```

File: src/popup/pages/Names/Claim.vue [hunk 2]
```diff
@@ -72,94 +73,99 @@ import CheckBox from '../../components/CheckBox.vue';
...
+            root.$store.dispatch('names/updatePointer', {
+              name: fullName,
               address: activeAccount.value.address,
             });
           }
-        } catch (e) {
-          openDefaultModal({
-            msg: e.message,
-          });
+        } catch (e: any) {
```
Comment (subhod-i) [inline src/popup/pages/Names/Claim.vue:148]: IDEM

**Review by subhod-i — CHANGES_REQUESTED** [review]

File: src/popup/pages/Names/Claim.vue (hunk 1 above)
Comment (CedrikNikita) [reply src/popup/pages/Names/Claim.vue:125]: Every other place in the project has `: any` in such case, i don't want to solve that in this particular issue.
```
Catch clause variable type annotation must be 'any' or 'unknown' if specified.Vetur(1196)
interface Error
```

Comment (subhod-i) [reply src/popup/pages/Names/Claim.vue:125]: Yeah, I see that. Perhaps we should start writing custom exceptions. This is not the right moment as too many refactorings already on it's way.

**Review by subhod-i — APPROVED** [review]

**Pull request merged**